firebase_admin/db.py

```diff
@@ -155,10 +155,10 @@ def parent(self):
 
     def build_headers(self, token=None):
         headers = {'content-type' : 'application/json; charset=UTF-8'}
-        if not token and self._client._session.credentials:
+        if not token and self._client._session.credentials: # pylint: disable=protected-access
```

Comment:
I think `self._client.session` should work here. `session` is exposed from the `HttpClient`:

https://github.com/firebase/firebase-admin-python/blob/c420b4f4e50b99ca62b7bc5c71e11d87219c4775/firebase_admin/_http_client.py#L53-L55

That way you won't need the linter directives at all.

Reply:
Oh, I didn't catch that earlier. I have changed it, thanks. I still have yet to write the tests. I'll finish the tests in a few days.

Reply:
great work @the-c0d3r . I am glad that this is finally progressing. Let me know if any help is required.